src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java

```diff
@@ -76,13 +78,21 @@ public ClaimantResponseContentProvider(
 
         switch (claimantResponse.getType()) {
             case ACCEPTATION: {
-                content.put("defendantAdmissionAccepted", "I accept this amount");
+                content.put("defendantAdmissionAccepted",
+                    String.format("I accept %s", getDefendantAdmissionStatus(claim, defendantResponse)));
                 ResponseAcceptation responseAcceptation = (ResponseAcceptation) claimantResponse;
-                content.putAll(responseAcceptationContentProvider.createContent(responseAcceptation));
+                content.putAll(responseAcceptationContentProvider.createContent(claim));
                 responseAcceptation.getFormaliseOption()
                     .map(FormaliseOption::getDescription)
                     .ifPresent(
-                        x -> content.put("formaliseOption", x)
+                        x -> {
```

Comment:
x is ok i guess but not sure...

Reply:
this was there from before - I should really change it to something meaningful